mobile/lib/ui/onboarding/onboarding_screen.dart

```diff
@@ -347,6 +358,19 @@ class _OnboardingPageWidget extends StatelessWidget {
     final screenHeight = MediaQuery.of(context).size.height;
     return screenHeight < _kSmallScreenHeightThreshold;
   }
+
+  String _composeBodyText() {
```

Comment:
Why do we do this instead of sending subtitle and description separately to the Widget?